python/hail/expr/aggregators/aggregators.py

```diff
@@ -1187,53 +1187,63 @@ def linreg(y, x, k_null=1) -> StructExpression:
 
     Notes
     -----
-    This aggregator returns a struct expression with ten fields:
+    The intercept must be included explicitly in `x` if desired.
```

Comment:
Maybe this should be a warning so it's super clear!

Reply:
Good idea, and I've also just made these warnings more consistent across all regressions.